packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -89,7 +89,8 @@ module.exports = (fileDetails, options) => {
 
   if (options.manifestTransforms) {
     if (Array.isArray(options.manifestTransforms)) {
-      manifestTransforms.concat(options.manifestTransforms);
+      manifestTransforms =
+          manifestTransforms.concat(options.manifestTransforms);
```

Comment:
@HenrikJoreteg thank you so much, only this change was holding this PR back... was about to work on it, u saved me some effort :)